ext/adaptors/ranges-lib.tex

```diff
@@ -592,7 +610,7 @@
 \effects Equivalent to:
 \begin{itemize}
 \item If \tcode{StoreSize} is \tcode{true},
-\tcode{subrange\{ranges::begin(r), ranges::end(r), ranges::size(r)\}}.
+\tcode{subrange\{ranges::begin(r), ranges::end(r), distance(r)\}}.
```

Comment:
Why?

Reply:
So the implementation isn't required to produce nasty conversion warnings when `ranges::size` returns an unsigned type.